about.html

```diff
@@ -0,0 +1,112 @@
+  <script src="https://kit.fontawesome.com/3848942bcd.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/about.css">
+</head>
+<body>
+  <header>
+    <div class="mylinks">
+      <ul class="link-menu">
+        <li class="link-item"><a ><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li class="link-item"><a ><i class="fa-brands fa-twitter"></i></a></li>
+        <li class="link-item"><a >English</a></li>
+        <li class="link-item"><a >My Page</a></li>
+        <li class="link-item"><a >Logout</a></li>
+      </ul>
+    </div>
+    <div class="menu-cont">
+      <i class="fa-sharp fa-solid fa-bars"></i>
+      <a href="index.html"><img src="images/logo.png" alt="Logo Concert Music Catholic" class="logo"></a>
+      <nav class="nav-menu">
+        <ul class="menu-dk">
+          <li><a class="menu-link" href="#">About</a></li>
+          <li><a class="menu-link" href="#">Program</a></li>
+          <li><a class="menu-link" href="#">Join</a></li>
+          <li><a class="menu-link" href="#">Sponsor</a></li>
+          <li><a class="menu-link" href="#">News</a></li>
+        </ul>
+        <button class="btn-campain"><a href="#">CMC World</a></button>
+      </nav>
+    </div>      
+  </header>
+  <div class="page">
+    <section class="principal">
+      <div class="tittle">
+        <h2 class="txt-tit-about">"United by the same Faith"</h2>
+        <h1 class="h1-about">CATHOLIC MUSIC CONCERT<br>WORLD TOUR July 2023</h1>
+        <p class="desc-about">A musical meeting of prayer and adoration with the best exponents of Catholic music. Three nights full of blessings, accompanied by teachings that will help us grow in our Faith.</p>
+        <div class="desc-email">
+          <p class="contact-about">Please contact us per Email for any futher questions about CMC World Tour 2023</p>
+          <a class="email" href="email://cmcworld2023@catholic.org">cmcworld2023@catholic.org</a> 
+        </div>        
+      </div>
+    </section>
+    <section class="logo">
+      <div class="logo-container">
+        <h2 class="tit-about">CATHOLIC MUSIC CONCERT<br>WORLD TOUR July 2023</h2>
+        <hr class="small-line">
+        <p class="logo-desc">The logo of CMC World Tour 2023 was decided trough the logo competition from 20.October to 22.September.</p>
+        <img class="logo-big" src="images/logo-big.png" alt="Logo CMC 2023">
+      </div>
+    </section>
+    <section class="past-events">
+      <div class="past-container">
+        <h2 class="tit-about">See the past CMC World Tour</h2>
+        <hr class="small-line">
+        <p class="logo-desc">Take a look at the two CMC World Tour wich tookin South America.</p>
+        <div class="past-img">
+          <img src="images/MCM2022.png" alt="Photo of the MCM 2022" class="past">
+        <img src="images/mcm2015.png" alt="Photo of the MCM 2015" class="past">
+        </div>          
+      </div>
+    </section>
```

Comment:
all sections in your about page width should be 100% taking up the full width of the screen as seen in the template design.

template design 👍🏾 
![image](https://user-images.githubusercontent.com/60612329/227383815-b5d45524-4ef8-4cf9-b59d-c6238c86e14b.png)

your design 
![image](https://user-images.githubusercontent.com/60612329/227384149-6e98441f-d803-401f-813e-7b16ae5c4075.png)

Reply:
@alexander16108, your observation was correct:

![image](https://user-images.githubusercontent.com/110146573/227437530-f77f7dba-dd94-4913-8f4c-718335554b18.png)